docs/source/contributing/index.md

```diff
@@ -0,0 +1,153 @@
+# Contributing
+
+PyMC-Marketing welcomes contributions from interested individuals or groups. These guidelines are provided to give potential contributors information to make their contribution compliant with the conventions of the PyMC-Marketing project, and maximize the probability of such contributions to be merged as quickly and efficiently as possible. Contributors need not be experts, but should be interested in the project, willing to learn, and share knowledge.
+
+There are 4 main ways of contributing to the PyMC-Marketing project (in ascending order of difficulty or scope):
+
+1. Submitting issues related to bugs or desired enhancements.
+2. Contributing or improving the documentation (docs) or examples.
+3. Fixing outstanding issues (bugs) with the existing codebase. They range from low-level software bugs to higher-level design problems.
+4. Adding new or improved functionality to the existing codebase.
+
+Items 2-4 require setting up a local development environment, see [Local development steps](#Contributing-code-via-pull-requests) for more information.
+
+## Opening issues
+We appreciate being notified of problems with the existing PyMC-Marketing code. We prefer that issues be filed the on [Github Issue Tracker](https://github.com/pymc-labs/pymc-marketing/issues), rather than on social media or by direct email to the developers.
+
+Please verify that your issue is not being currently addressed by other issues or pull requests by using the GitHub search tool to look for key words in the project issue tracker.
+
+## Contributing code via pull requests
+
+While issue reporting is valuable, we strongly encourage users who are inclined to do so to submit patches for new or existing issues via pull requests. This is particularly the case for simple fixes, such as typos or tweaks to documentation, which do not require a heavy investment of time and attention.
+
+Contributors are also encouraged to contribute new code to enhance PyMC-Marketing's functionality, via pull requests.
+
+The preferred workflow for contributing to PyMC-Marketing is to fork the GitHub repository, clone it to your local machine, and develop on a feature branch.
+
+For more instructions see the [Pull request checklist](#pull-request-checklist)
+
+## Local development steps
+
+1. If you have not already done so, fork the [project repository](https://github.com/pymc-labs/pymc-marketing) by clicking on the 'Fork' button near the top right of the main repository page. This creates a copy of the code under your GitHub user account.
+
+1. Clone your fork of the `pymc-marketing` repo from your GitHub account to your local disk, and add the base repository as a remote:
+
+    ```bash
+    git clone git@github.com:<your GitHub handle>/pymc-marketing.git
+    cd pymc-marketing
+    git remote add upstream git@github.com:pymc-labs/pymc-marketing.git
```

Comment:
If you use the `GitHub` CLI, then the required command is just `gh repo fork pymc-labs/pymc-marketing` 😄

Reply:
Added in https://github.com/pymc-labs/pymc-marketing/pull/1023/commits/efc8296d5665943e446c0c0d8b3c7cf2f6583760